### PR TITLE
fix: Azure storage and Vault simplified

### PR DESCRIPTION
**Pull request by zacharyblasczyk**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -129,21 +126,6 @@ module "app_aks" {
...
-  queue_name      = try(module.storage[0].queue.name, "")
-  blob_container  = var.external_bucket == null ? coalesce(var.blob_container, local.container_name) : ""
-  storage_account = var.external_bucket == null ? coalesce(var.storage_account, local.account_name) : ""
-  storage_key     = var.external_bucket == null ? coalesce(var.storage_key, local.access_key) : ""
-  bucket          = "az://${local.storage_account}/${local.blob_container}"
-  queue           = (var.use_internal_queue || var.blob_container == "" || var.external_bucket == null) ? "internal://" : "az://${local.account_name}/${local.queue_name}"
-
-  redis_connection_string = "redis://:${module.redis.instance.primary_access_key}@${module.redis.instance.hostname}:${module.redis.instance.port}"
-}
-
```
Comment (zacharyblasczyk) [inline main.tf:146]: @jsbroks I am so glad to get rid of this.

**Comment by zacharyblasczyk** [comment]
[This](https://github.com/wandb/helm-charts/pull/199#pullrequestreview-2216280306) needs to be released before we can merge and roll out to QA

**Review by jsbroks — APPROVED** [review]

**Comment by zacharyblasczyk** [comment]
## Migrations

### Upgrading from 2.x -> 3.x

`3.x` of this terraform requires helm version `0.17.x`+ to function properly if you don't set 
- `var.storage_account`
- `var.blob_container `
- `var.storage_key`

**Review by danielpanzella — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 2.13.1 :tada:

**Pull request merged**